### DISCUSSION TITLE
Using nocodb-seed /git method fine. But where is my nocoDB metadata?

### DISCUSSION THREAD

**Discussion by philipackerley** [nocodb/nocodb#677] [Q&A] [answered]
Hi, Basically I built nocoDB client using the git/nocodb-seed instructions in a vanilla fashion. I was then able to connect to a mySQl db project and actually entered my own READ only user account name/pw but connercting to another user's schema , all connected fine via localhost::8080  and it listed the 6 tables in that other user's schema. ?

 Puzzled why did this actually work as  I have no write access to that schema? Where is the metadata/project information stored for my instance of nocoDB? I am on a windows 10 machine.  I stopped and restarted the localhost server, it still works? I need to document for use by other users and point out to them where their metadata lives, thanks.

**Comment by o1lab** [comment, marked as answer]
It will be stored as a sqlite file in root folder of project directory. Search for noco.db

For using it in production see [here](https://docs.nocodb.com/getting-started/installation#production-setup), you can use NC_DB variable to point to a mysql database.

**Reply by philipackerley** [reply]
Thanks! I was overthinking it, I thought must be sqllite , and then saw the subfolder with the .db file, but I did not relate a filetype .db to sqlite at first. cheers anyway, working great BTW.
